internal/langserver/handlers/command/terraform.go

```diff
@@ -0,0 +1,56 @@
+package command
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/creachadair/jrpc2/code"
+	"github.com/hashicorp/terraform-ls/internal/langserver/cmd"
+	"github.com/hashicorp/terraform-ls/internal/uri"
+)
+
+const terraformVersionRequestVersion = 0
+
+type terraformInfoResponse struct {
+	FormatVersion     int    `json:"v"`
+	RequiredVersion   string `json:"required_version,omitempty"`
+	DiscoveredVersion string `json:"discovered_version,omitempty"`
+	DiscoveredPath    string `json:"discovered_path,omitempty"`
```

Comment:
```suggestion
```
Shall we omit this for now, until we figure out how to persist it and keep it up to date?